### DISCUSSION TITLE
"Flight Control" tab extremely laggy in cfclient

### DISCUSSION THREAD

**Discussion by machacekch** [bitcraze/discussions#1437] [Q&A]
When connected, cfclient becomes very unresponsive on the "Flight Control" tab.
I'm using the LH deck and the "Command Based Flight Control". Sometimes it takes the GUI 10+ seconds to register any button press (such as switching to another tab or hitting the "Emergency stop"). The artificial horizon is also delayed by multiple seconds.

What's going on here? Is there something I can disable to make it more responsive? It's bad on both Linux and Win10.

**Comment by knmcguire** [comment]
Hi! This might be due to the graphics card used or if there are any other heavy graphical programs running on the background. This is also to be shown to happen in the virtual machine as well I'm afraid.

If you try it on a different PC do you have the same issue? Perhaps your laptop is throttling the graphic performance of your laptop to be more efficient (I had that problem myself).
